layouts/partials/webinar-list-item.html

```diff
@@ -9,7 +9,7 @@
             {{ else }}
             <a class="text-2xl" href="/webinars/{{ .Params.url_slug }}/">
             {{ end }}
-                <img class="w-full" src="{{ .Params.preview_image }}" />
+                <img class="w-full h-full" src="{{ .Params.preview_image }}" />
```

Comment:
This is scaling the image in ways that are making some internet pandas cry. I'll increase the image size to make it work. 😄

Reply:
:)

Reply:
let me know if I need to ask for a different image...